sssom/context.py

```diff
@@ -36,6 +52,13 @@ def get_built_in_prefix_map() -> PrefixMap:
 def add_built_in_prefixes_to_prefix_map(
     prefix_map: Optional[PrefixMap] = None,
 ) -> PrefixMap:
+    """Add built-in prefix map from the sssom_context variable in the auto-generated 'internal_context.py' file.
+
+    [Auto generated from sssom.yaml by jsonldcontextgen.py]
+
+    :param prefix_map: Prefix map, defaults to None
```

Comment:
obviously it defaults to none, that's the default. what happens if None is passed?

Reply:
@matentzn